tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -106,7 +106,7 @@ def process_sqs_messsage(self, message):
             self.send_resource_owner_messages(data)
         if 'event-owner' in targets:
             targets.remove('event-owner')
-            recipient = self.get_aws_username_from_event(data['event'])
+            recipient = self.get_aws_username_from_event(data['event'], "userName")
```

Comment:
we probably need to make this configurable, the exact field tends to be dependent on org specific auth setups it seems.